appinfo/routes.php

```diff
@@ -25,6 +25,8 @@
 	'routes' => [
 		['name' => 'page#index', 'url' => '/', 'verb' => 'GET'],
 		['name' => 'page#index', 'url' => '/{group}', 'verb' => 'GET', 'postfix' => 'group'],
-		['name' => 'page#index', 'url' => '/{group}/{contact}', 'verb' => 'GET', 'postfix' => 'group.contact']
+		['name' => 'page#index', 'url' => '/{group}/{contact}', 'verb' => 'GET', 'postfix' => 'group.contact'],
+		['name' => 'social_api#update_contact', 'url' => '/api/v1/social/avatar/{network}/{addressbookId}/{contactId}', 'verb' => 'GET'],
+		['name' => 'social_api#set_app_config', 'url' => '/api/v1/social/config/{key}', 'verb' => 'POST'],
```

Comment:
I'm pretty sure you can use camel case on both the controller name as well as the method

Reply:
it's described like that in the [developers guide](https://docs.nextcloud.com/server/latest/developer_manual/basics/routing.html). should I change it to camel case?

Reply:
TIL! In that case it's probably fine :shrug: I'm the one who used it wrong apparently :stuck_out_tongue: